corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -454,12 +454,12 @@ def test_datasource_change_triggers_change_signal(self, datetime_mock, create_re
         create_repeat_records_mock.assert_called()
         # Assert that it will be created with the expected args
         call_args = create_repeat_records_mock.call_args[0]
-        self.assertEqual(call_args[0], DataSourceRepeater)
-        self.assertTrue(isinstance(call_args[1], DataSourceUpdate))
-        update_log = call_args[1]
-        self.assertEqual(update_log.domain, self.domain)
-        self.assertEqual(update_log.data_source_id, self.config._id)
-        self.assertEqual(update_log.doc_id, sample_doc["_id"])
+        assert call_args[0] == DataSourceRepeater
+        datasource_update = call_args[1]
+        assert type(datasource_update) is DataSourceUpdate
+        assert datasource_update.domain == self.domain
+        assert datasource_update.data_source_id == uuid.UUID(self.config._id)
+        assert datasource_update.doc_ids == [sample_doc["_id"]]
```

Comment:
Just curious what inspired this change?

Reply:
I'm not sure what part of this change you're referring to, so I'll just talk about all of it.

Previously, `update_log.doc_id` stored only one doc ID. This change updates the test to verify that `datasource_update.doc_ids` is a list containing the ID of `sample_doc`.

The change also uses the [pytest convention](https://docs.pytest.org/en/stable/how-to/assert.html) of Pythonic `assert` statements instead of [JUnit-style](https://junit.org/junit5/docs/current/user-guide/) `assertFoo()` methods (which exist because at the time that JUnit was created, Java didn't have an `assert` statement -- it does now).